lona/app.py

```diff
@@ -428,8 +430,8 @@ def run(
 
         # setup arguments
         server_args = Namespace(
-            host='localhost',
-            port=8080,
+            host=os.environ.get('LONA_DEFAULT_HOST', 'localhost'),
+            port=int(os.environ.get('LONA_DEFAULT_PORT', 8080)),
```

Comment:
We will need a check here if the value of `LONA_DEFAULT_PORT` can be type casted to an int. This will crash into a cryptic traceback if not